src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/00 About scenarios/03 Dropped iterations.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Dropped iterations
+excerpt: Explanations about how your scenario configuration or SUT performance can lead to dropped iterations
+---
+
+Sometimes, a scenario can't run the expected number of iterations.
+k6 tracks the number of unsent iterations in a counter metric, `dropped iterations`.
+The number of dropped iterations can be valuable data when you debug executors or analyze results.
+
+Dropped iterations usually happen for one of two reasons:
+- The executor configuration is insufficient.
+- The SUT can't handle the configured VU arrival rate.
+
+### Configuration-related iteration drops
+
+Dropped iterations happen for different reasons in different types of executors.
+
+With `shared-iterations` and `per-vu-iterations`, iterations drop if the scenario reaches its `maxDuration` before all iterations finish.
+To mitigate this, you likely need to increase the value of the duration.
+  
+With `constant-arrival-rate` and `ramping-arrival-rate`, iterations drop if there are no free VUs.
+**If it happens at the beginning of the test, you likely just need to allocate more VUs.**
+If this happens later in the test, the dropped iterations might happen because SUT performance is degrading and iterations are taking longer to finish.
+
+### SUT-related iteration drops
+
+At a certain point of high latency or longer iteration durations, k6 can no longer send VUs at the configured rate.
+As a result, the executor will drop iterations.
+
+The reasons for these dropped iterations vary:
+- The SUT response has become so long that k6 starts dropping scheduled VUs from the queue.
+- The SUT iteration duration has become so long that k6 needs to schedule more VUs to reach the target arrival rate, exceeding the number of scheduled iterations.
+- Some network errors between the generator and the SUT have caused iterations to drop.
```

Comment:
```suggestion
```

This is not a thing that happens :sweat_smile: it might lead to interrupted iterations (e.g. an iteration that was already started but need to be stopped because we ran out of time), but we don't track these yet :disappointed: